aomi/cli.py

```diff
@@ -137,33 +155,36 @@ def action_runner(operation):
         sys.exit(0)
 
     client = aomi.vault.client(operation, opt)
-    if operation == 'extract_file':
-        if len(args) == 3:
-            aomi.render.raw_file(client, args[1], args[2], opt)
-            sys.exit(0)
-    elif operation == 'environment':
-        if len(args) >= 2:
-            paths = args[1:]
-            aomi.render.env(client, paths, opt)
-            sys.exit(0)
-    elif operation == 'aws_environment':
-        if len(args) == 2:
-            aomi.render.aws(client, args[1], opt)
-            sys.exit(0)
-    elif operation == 'seed':
-        if len(args) == 1:
-            aomi.validation.gitignore(opt)
-            aomi.vault.seed(client, opt)
-            sys.exit(0)
-    elif operation == 'template':
-        if len(args) >= 4:
-            paths = args[3:]
-            aomi.render.template(client, args[1], args[2], paths, opt)
-            sys.exit(0)
-    elif operation == 'token':
-        if len(args) == 1:
-            print(client.token)
-            sys.exit(0)
+    if operation == 'extract_file' and len(args) == 3:
```

Comment:
Not against this, but it's outside the scope of this PR. Create an issue to track this discussion. The cli module is definitely getting a little unwieldily.